moredata/models/merging_data.py

```diff
@@ -0,0 +1,17 @@
+import geopandas
+class MergingData:
+
+    @classmethod
+    def from_geodataframe(self,geodataframe):
+        self.data = geopandas.read_file(geopandas.datasets.get_path(geodataframe))
+        return self.data
+
+    @classmethod
+    def from_path(self, path):
+        self.data = geopandas.read_file(path) 
+        return self.data
+
+    @classmethod
+    def spatial_join(self, gdf_enricher ,gdf_enrichering,how, predicate ):
```

Comment:
This function shouldn't exist because we don't want to container the geopandas sjoin function in our package.